sample/common/src/commonMain/kotlin/com/trikot/viewmodels/sample/viewmodels/home/SwitchesViewModel.kt

```diff
@@ -15,7 +15,7 @@ class SwitchesViewModel(navigationDelegate: NavigationDelegate) :
         MutableHeaderListItemViewModel("Switch"),
         MutableToggleSwitchListItemViewModel().apply {
             toggleSwitch.action = ViewModelAction {
-                println(toggleSwitch.isOn.value)
+                println(toggleSwitch.isOn)
```

Comment:
Is it a common thing to add logs throught println like that in trikot?

Reply:
Nop it's in the sample app. But not sure if it should be there